Gotify.json

```
@@ -0,0 +1,39 @@
+{
+    "Gotify": {
+        "description": "A simple server for sending and receiving messages. <p>Based on the official Gotify x86 docker image: <a href='https://hub.docker.com/r/gotify/server' target='_blank'>https://hub.docker.com/r/gotify/server</a>.</p>",
```

Harmonize description with current guidelines:
```suggestion
        "description": "A simple server for sending and receiving messages. <p>Based on the official Gotify Docker image: <a href='https://hub.docker.com/r/gotify/server' target='_blank'>https://hub.docker.com/r/gotify/server</a> available for x86_64 architecture only.</p>",
```